exporters/elasticsearch-exporter/src/main/resources/zeebe-record-process-instance-modification-template.json

```diff
@@ -0,0 +1,57 @@
+{
+  "index_patterns": [
+    "zeebe-record_process-instance-modification_*"
+  ],
+  "composed_of": ["zeebe-record"],
+  "priority": 20,
+  "version": 1,
+  "template": {
+    "settings": {
+      "number_of_shards": 1,
+      "number_of_replicas": 0,
+      "index.queries.cache.enabled": false
+    },
+    "aliases": {
+      "zeebe-record-process-instance-modification": {}
+    },
+    "mappings": {
+      "properties": {
+        "value": {
+          "dynamic": "strict",
+          "properties": {
+            "processInstanceKey": {
+              "type": "long"
+            },
+            "terminateInstructions": {
+              "properties": {
+                "elementInstanceKey": {
+                  "type": "long"
+                }
+              }
+            },
+            "activateInstructions": {
+              "properties": {
+                "elementId": {
+                  "type": "keyword"
+                },
+                "ancestorScopeKey": {
+                  "type": "long"
+                },
+                "variableInstructions": {
+                  "properties": {
+                    "variables": {
+                      "enabled": false
```

Comment:
I guess it's not necessary because it's disabled (so discarded by ES). But, I'm no expert. This is just copied from [zeebe-record-process-instance-creation-template.json#L34-L36](https://github.com/camunda/zeebe/blob/main/exporters/elasticsearch-exporter/src/main/resources/zeebe-record-process-instance-creation-template.json#L34-L36)